test/Microsoft.AspNetCore.SignalR.Redis.Tests/RedisHubLifetimeManagerTests.cs

```diff
@@ -481,6 +484,88 @@ public class RedisHubLifetimeManagerTests
             }
         }
 
+        [Fact]
+        public async Task WritingToRemoteConnectionThatFailsDoesNotThrow()
+        {
+            var manager1 = new RedisHubLifetimeManager<MyHub>(new LoggerFactory().CreateLogger<RedisHubLifetimeManager<MyHub>>(), Options.Create(new RedisOptions()
+            {
+                Factory = t => new TestConnectionMultiplexer()
+            }));
+            var manager2 = new RedisHubLifetimeManager<MyHub>(new LoggerFactory().CreateLogger<RedisHubLifetimeManager<MyHub>>(), Options.Create(new RedisOptions()
+            {
+                Factory = t => new TestConnectionMultiplexer()
+            }));
+
+            using (var client = new TestClient())
+            {
+                // Force an exception when writing to connection
+                var output = new Mock<Channel<HubMessage>>();
+                output.Setup(o => o.Out.WaitToWriteAsync(It.IsAny<CancellationToken>())).Throws(new Exception());
+
+                var connection = new HubConnectionContext(output.Object, client.Connection);
+
+                await manager2.OnConnectedAsync(connection).OrTimeout();
+
+                await manager1.InvokeConnectionAsync(connection.ConnectionId, "Hello", new object[] { "World" }).OrTimeout();
```

Comment:
add a comment explaining why that it does not fail because WaitToWriteAsync is not even invoked - rather we publish to  Redis.